contracts/governance/Managed.sol

```diff
@@ -28,6 +28,14 @@ contract Managed {
     mapping(bytes32 => address) private addressCache;
     uint256[10] private __gap;
 
+    // Immutables
```

Comment:
Is it better to be immutable than const? as there are no parametrization happening

Reply:
I think in this case it's the same as we are using `bytes32` to store the result (https://docs.soliditylang.org/en/v0.8.11/contracts.html#constant-and-immutable-state-variables). 

I went with `immutable` because the reports suggested it, but should be the same as far as I understand.